src/components/LmsConfigurations/index.test.jsx

```diff
@@ -65,11 +77,12 @@ describe('<LmsConfigurations /> ', () => {
   it('return Fragment/Collapsible components when all have 404', async () => {
     LmsApiService.fetchMoodleConfig.mockRejectedValue(notFoundMoodleResponse);
     LmsApiService.fetchCanvasConfig.mockRejectedValue(notFoundCanvasResponse);
+    LmsApiService.fetchBlackboardConfig.mockRejectedValue(notFoundBlackboardResponse);
```

Comment:
Same as above.

Reply:
Again, assuming this is in reference to individual tests for this, I am again disagreeing, but this time because it actually just functionally does not work.

We *only* return a list of forms if *every* config 404s. There is nothing to test individually here. If you were successful on any one of them, we'd return that component. If any of them fail, we return the error page. It's only if all have 404s that we show the list which is what this is testing.

Reply:
We will be essentially testing "individual" 404s by doing a successful config test for each config (because the others would have to be 404s instead of other error codes)